contrib/intel/jenkins/summary.py

```diff
@@ -700,6 +700,51 @@ def check_line(self, line):
             self.check_fail(line)
             self.check_exclude(line)
 
+class DmabufSummarizer(Summarizer):
+    def __init__(self, logger, log_dir, prov, file_name, stage_name):
+        super().__init__(logger, log_dir, prov, file_name, stage_name)
+
+        self.test_type = ''
+        self.node = "pvc[6,8]"
+
+    def check_type(self, line):
+        if "Running" in line:
+            self.test_type = line.split()[2]
+
+    def check_line(self, line, log_file):
+        self.check_type(line)
+
+    def check_name(self, line):
+        if "client_command" in line:
+            name_list = line.split()[-2:]
+            if "ssh pvc6" in line:
```

Comment:
please use a variable for this. the nodes won't be the same every time in the future.